bugbug/tools/code_review.py

```diff
@@ -152,7 +153,7 @@ class LargeDiffError(Exception):
 - include praising;
 - ask if changes are intentional or ask to ensure things exist.
 
-Do not report any explanation about your choice. Only return a valid JSON list.
+Only return a valid JSON list. Do not drop any key from the JSON objects.
```

Comment:
Is the second part needed?

Reply:
At the beginning, it was without the sound part. However, the filtering step was omitting the `explanation` key from he final result.